app/search_builders/curation_concerns/file_set_search_builder.rb

```diff
@@ -2,9 +2,9 @@ module CurationConcerns
   class FileSetSearchBuilder < ::SearchBuilder
     include CurationConcerns::SingleResult
 
-    # This overrides the filter_models in FilterByType
-    def filter_models(solr_parameters)
-      solr_parameters[:fq] << ActiveFedora::SolrQueryBuilder.construct_query_for_rel(has_model: ::FileSet.to_class_uri)
+    # This overrides the models in FilterByType
+    def models
```

Comment:
The problem is that `to_class_uri` doesn't return a uri. It simply returns a model name.  This is a vestige from the Fedora 3 days.